src/projectscene/qml/Audacity/ProjectScene/toolbars/UndoRedoToolBar.qml

```diff
@@ -0,0 +1,40 @@
+/*
+* Audacity: A Digital Audio Editor
+*/
+import QtQuick
+
+import Muse.UiComponents
+
+import Audacity.ProjectScene
+
+StyledToolBarView {
+    navigationPanel.name: "UndoRedoToolBar"
+    navigationPanel.accessible.name: qsTrc("projectscene", "Undo/redo toolbar")
+
+    spacing: 0
+    //! TODO AU4: apply padding when StyledToolBarView will have padding property added
+    // padding: 6
+
```

Comment:
Please add rowHeight: 28
In the screenshot, you can see how the toolbar overlaps the strip below:
![image](https://github.com/user-attachments/assets/e32e37b4-be85-4027-972e-3402978caef7)